client/src/components/standing_table/StandingTable.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { FunctionComponent } from 'react';
+import Box from '@material-ui/core/Box';
+import { withTheme } from '@material-ui/core';
+import styled from 'styled-components';
+import Divider from '@material-ui/core/Divider';
+import Paper from '@material-ui/core/Paper';
+import map from 'lodash/map';
+import find from 'lodash/find';
+
+import { shortenName } from '../../utils/format';
+import { TableType } from '../../types/table_type';
+
+import { TableRow } from './standingData';
+
+interface StandingTable {
+  className?: string;
+  title?: string;
+  tableRowData: TableRow[] | null;
+  tableHeaderData: string[];
+  headerLongField: string[];
+  rowLongField: string[];
+  cusElevation?: number;
```

Comment:
why do you need this?` cusElevation`

Reply:
I used a `standing Table` to display each match on the `Schedule & Score` tab under `league` page. so I tried to remove the `paper shadow` using an elevation value is 0.
Btw, I changed the name to `elevation`